website/docs/using-odo/json-output.md

```diff
@@ -0,0 +1,43 @@
+---
+title: JSON Output
+sidebar_position: 2
```

Comment:
We're planning to put Spring Boot + Postgres in the "Using odo" section such that it'll show the complete flow starting from creating a component to creating URL, storage, services, and linking with services, components, etc. In that context, putting JSON Output doc under "Using odo" seems out of place to me.

For a few guides that we're importing from old website which don't make sense to be put in "Using odo" section, we are putting them under "Tutorials". A PR is open for it #4934.

I'm inclined to put this under "Tutorials" as well. What do you feel?

Reply:
I would prefer to place this page under a "Reference" section

Reply:
Yes, that makes a lot of sense. Where do you think we could put up that section? Current sections:
- Introduction (no sub-sections)
- Getting Started
- Using odo
- Command Reference
- Tutorials
- Architecture
- Contributing to odo

Reply:
This page could appear at the end of the **Command Reference** section

Reply:
Ack. 👍🏾

Reply:
@feloy can you create a new page and move this to the `Command Reference` section, or do you intend to do this in another PR?